cmd/instance_create.go

```diff
@@ -209,6 +216,30 @@ func (c *instanceCreateCmd) cmdRun(_ *cobra.Command, _ []string) error { //nolin
 				return
 			}
 		}
+
+		if c.BlockStorageVolume != "" {
+			volumes, e := clientv3.ListBlockStorageVolumes(ctx)
```

Comment:
I prefer to use `err2`, `err3` and so on instead of `e`.

Reply:
Yes why not. In Golang, declare short and use short is still in the idiom.
But let's apply the proposal ;)